lib/amazing_print/ext/mongo_mapper.rb

```diff
@@ -90,7 +90,7 @@ def awesome_mongo_mapper_instance(object)
         label = "#{colorize('embedded', :assoc)} #{label}"
       end
 
-      "#{label} " << awesome_hash(data)
+      [label, awesome_hash(data)].join(' ')
```

Comment:
If the String should stay mutable, this should work too: `+"#{label} #{awesome_hash(data)}"`. Up to you which one's more readable.

Reply:
I went back and forth on this one but based on #41 I thought using the same approach everywhere was nice.